### PR TITLE
RAP-4269: Initial PR for Add StructuredStore CRUD operations to REST …

### PR DESCRIPTION
**Pull request by jonathan-major**
…Server

### CODE REVIEW

File: Apps/RestServer/README.md [hunk 1]
```diff
@@ -55,8 +55,9 @@ You can also obtain an api key by making a direct call to the Rapture User API u
 * [Blob](#blob) 
 * [Series](#series)
 * [Workflow](#workflow)
+* [StructuredStore](#sstore)
 
-<a name="document"/>
+<a name="document""/>
```
Comment (dukenguyen) [inline Apps/RestServer/README.md]: typo?

File: Apps/RestServer/src/main/java/rapture/server/rest/RestServer.java [hunk 2]
```diff
@@ -236,9 +239,125 @@ private void setupRoutes() {
...
+            if (Kernel.getStructured().structuredRepoExists(ctx, authority)) {
+                halt(409, String.format("Repo [%s] already exists", authority));
+            }
+            Kernel.getStructured().createStructuredRepo(ctx, authority, config);
+            return new RaptureURI(authority, Scheme.STRUCTURED).toString();
+        });
+        
+        post("/sstore/:authority/:table", (req, res) -> {
+        	log.info(req.body());
+        	Map<String, Object> data = JacksonUtil.getMapFromJson(req.body());
```
Comment (dukenguyen) [inline Apps/RestServer/src/main/java/rapture/server/rest/RestServer.java]: indentation off.  I recommend auto-formatting on save.

Go to Window->Preferences->Java->Editor->SaveActions and check the Format source code option. Then your source code will be formatted automatically each time you save it.

File: Apps/RestServer/README.md (hunk 1 above)
Comment (jonathan-major) [inline Apps/RestServer/README.md]: fixed it

File: Apps/RestServer/src/main/java/rapture/server/rest/RestServer.java (hunk 2 above)
Comment (jonathan-major) [inline Apps/RestServer/src/main/java/rapture/server/rest/RestServer.java]: I was missing the incapture style xml. so loaded that and then update the save actions setting.

**Pull request merged**